viewing_party/main.py

```diff
@@ -0,0 +1,151 @@
+#Wave 1 
+def create_movie(title,genre,rating): 
+    if (title and genre and rating): 
+        movie = {}
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
+    elif not title or not genre or not rating:
+        return None
+  
+def add_to_watched(user_data,movie): 
+    user_data["watched"].append(movie)
+    return user_data 
+
+def add_to_watchlist(user_data,movie):
+    if movie not in user_data["watchlist"]: 
+        user_data["watchlist"].append(movie)
+    return user_data 
+
+def watch_movie(user_data, title):
+
+    for e in user_data["watchlist"]:
+        if e["title"] == title:
+            user_data["watchlist"].remove(e) 
+            user_data['watched'].append(e)
+    return user_data 
+ 
+
+# Wave # 2
+
+def get_watched_avg_rating(user_data): 
+    watched = user_data["watched"]
+    if len (watched) == 0:  
```

Comment:
Style: Avoid spaces between function names and the argument list:

```suggestion
    if len(watched) == 0:  
```